### PR TITLE
update quickstart, make schema guide more single repo friendly

### PR DESCRIPTION
**Pull request by philmillman**

### CODE REVIEW

File: packages/docs-site/src/content/docs/docs/guides/schema.mdx [hunk 1]
```diff
@@ -89,6 +89,8 @@ export default defineDmnoWorkspace({
 });
 ```
 
+> If you're not in a monorepo, you can skip to [Defining config items](/docs/guides/schema/#defining-config-items/).
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/guides/schema.mdx:92]: extra trailing slash breaking the build!

File: packages/docs-site/src/content/docs/docs/guides/schema.mdx [hunk 2]
```diff
@@ -18,7 +18,19 @@ You'll always have a `.dmno/config.mts` file at the root of your project, and if
 A typical DMNO project structure:
 
 <Tabs syncKey='repoType'>
-  <TabItem label="Monorepo">
+  <TabItem label="Single repo">
+    <FileTree>
+        - / (root of your project)
+          - .dmno/ (your workspace config)
+            - **config.mts**
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/guides/schema.mdx:25]: maybe add all the files on this one, and then on the monorepo case, just include the .dmno folder itself?

File: packages/docs-site/src/content/docs/docs/get-started/quickstart.mdx [hunk 3]
```diff
@@ -14,43 +14,66 @@ import TabbedCode from '@/components/TabbedCode.astro';
...
+      isRoot: true,
+      schema: {
+        PUBLIC_API_BASE_URL: {
+          extends: DmnoBaseTypes.url,
+          description: 'Base URL for the public API',
+        },
+        PUBLIC_GOOGLE_ANALYTICS_ID: {
+          description: 'Google Analytics ID',
+        },
+        DATABASE_URL: {
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/get-started/quickstart.mdx:53]: should probably be sensitive

File: packages/docs-site/src/content/docs/docs/guides/schema.mdx (hunk 2 above)
Comment (philmillman) [reply packages/docs-site/src/content/docs/docs/guides/schema.mdx:25]: yeah good call

**Pull request merged**